ci_support/run_docker_build.sh

```diff
@@ -37,8 +37,6 @@ echo "$config" > ~/.condarc
 # A lock sometimes occurs with incomplete builds. The lock file is stored in build_artefacts.
 conda clean --lock
 
-# library dependencies for matplotlib
-yum install -y libXext libXrender libSM tk libX11-devel
```

Comment:
Nearly done with a new Docker image which will hopefully fix this an a bunch of other related issues.